e2e/test/scenarios/metrics/reproductions/metrics-reproductions.cy.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import {
+  createQuestion,
+  getNotebookStep,
+  main,
+  restore,
+} from "e2e/support/helpers";
+
+const { ORDERS_ID, ORDERS } = SAMPLE_DATABASE;
+
+describe("issue 47058", () => {
+  before(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.intercept("GET", "/api/card/*/query_metadata", req =>
+      req.continue(() => new Promise(resolve => setTimeout(resolve, 1000))),
+    ).as("metadata");
+
+    createQuestion({
+      name: "Metric 47058",
+      type: "metric",
+      query: {
+        "source-table": ORDERS_ID,
+        aggregation: [["count"]],
+      },
+    }).then(({ body: { id: metricId } }) => {
+      createQuestion({
+        name: "Question 47058",
+        type: "question",
+        query: {
+          "source-table": ORDERS_ID,
+          fields: [
+            ["field", ORDERS.ID, {}],
+            ["field", ORDERS.TOTAL, {}],
+          ],
+          aggregation: [["metric", metricId]],
+          limit: 1,
+        },
+      }).then(({ body: { id: questionId } }) => {
```

Comment:
`createQuestion` supports second parameter `{visitQuestion: true}`, but then you'll need to call `openNotebook` helper to open notebook

Reply:
That `visitQuestion` waits for the metadata to load, which is what we want to test in this PR so we can't use that.